src/process_utility.c

```diff
@@ -2975,7 +2975,7 @@ process_create_table_end(Node *parsetree)
 }
 
 static inline const char *
-typename_get_unqual_name(TypeName *tn)
+typename_get_unqualified_name(TypeName *tn)
```

Comment:
There are some fairly verbose tokens in this file, I don't see any particular reason to abbreviate this...